platform/registry.go

```diff
@@ -102,108 +102,116 @@ var (
 	case coin.Litecoin().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.LTC})
+		return &bitcoin.Platform{CoinIndex: coin.LTC}
 	case coin.Bitcoincash().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.BCH})
+		return &bitcoin.Platform{CoinIndex: coin.BCH}
 	case coin.Dash().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.DASH})
+		return &bitcoin.Platform{CoinIndex: coin.DASH}
 	case coin.Doge().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.DOGE})
+		return &bitcoin.Platform{CoinIndex: coin.DOGE}
 	case coin.Zcash().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.ZEC})
+		return &bitcoin.Platform{CoinIndex: coin.ZEC}
 	case coin.Zcoin().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.XZC})
+		return &bitcoin.Platform{CoinIndex: coin.XZC}
 	case coin.Viacoin().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.VIA})
+		return &bitcoin.Platform{CoinIndex: coin.VIA}
 	case coin.Ravencoin().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.RVN})
+		return &bitcoin.Platform{CoinIndex: coin.RVN}
 	case coin.Qtum().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.QTUM})
+		return &bitcoin.Platform{CoinIndex: coin.QTUM}
 	case coin.Groestlcoin().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.GRS})
+		return &bitcoin.Platform{CoinIndex: coin.GRS}
 	case coin.Zelcash().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.ZEL})
+		return &bitcoin.Platform{CoinIndex: coin.ZEL}
 	case coin.Decred().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.DCR})
+		return &bitcoin.Platform{CoinIndex: coin.DCR}
 	case coin.Digibyte().Handle:
-		platformList = append(platformList, &bitcoin.Platform{CoinIndex: coin.DGB})
+		return &bitcoin.Platform{CoinIndex: coin.DGB}
 	case coin.Nebulas().Handle:
-		platformList = append(platformList, &nebulas.Platform{})
+		return &nebulas.Platform{}
 	case coin.Fio().Handle:
-		platformList = append(platformList, &fio.Platform{})
+		return &fio.Platform{}
 	case coin.Algorand().Handle:
-		platformList = append(platformList, &algorand.Platform{})
+		return &algorand.Platform{}
 	case coin.Nano().Handle:
-		platformList = append(platformList, &nano.Platform{})
+		return &nano.Platform{}
 	case coin.Harmony().Handle:
-		platformList = append(platformList, &harmony.Platform{})
+		return &harmony.Platform{}
 	case coin.Kusama().Handle:
-		platformList = append(platformList, &polkadot.Platform{CoinIndex: coin.KSM})
+		return &polkadot.Platform{CoinIndex: coin.KSM}
 	default:
-		platformList = allPlatformsList
+		return nil
 	}
+}
 
+func getActivePlatforms(platformHandle string) []blockatlas.Platform {
+	var platformList []blockatlas.Platform
```

Comment:
```suggestion

	logger.Info("Loaded with: ", logger.Params{"handle": platformHandle})
	
	platform := getPlatform(platformHandle)
	if platform != nil {
		return []{platform}
	} else {
		return allPlatformsList
	}
```

Did you consider simpler version?